chain/client/src/chunks_delay_tracker.rs

```diff
@@ -46,7 +46,7 @@ impl ChunksDelayTracker {
 
     // Computes the delay between receiving a block and receiving the latest of its chunks.
     // Updates the metric to the max delay across the most recently processed blocks.
-    fn update_chunks_metric(&mut self, head_height: BlockHeight) {
+    fn get_max_delay_us(&mut self, head_height: BlockHeight) -> u128 {
```

Comment:
It may be a good idea to use `Duration`.

Reply:
I see later we actually compare this to `Duration`
Instead of using:
```
  assert_eq!(tracker.get_max_delay_us(head_height), Duration::from_secs(5).as_micros());
```
we could make it
```
```
  assert_eq!(tracker.get_max_delay_us(head_height), Duration::from_secs(5));
```